platform/brains2/src/lib/leds.cpp

```diff
@@ -151,7 +141,8 @@ void setBrightness(int brightness) {
   _brightness = brightness;
 }
 void show(const Pixel *const pixels, const int pixel_count) {
-  while(!show_pixels(pixels, pixel_count));
+  while (!show_pixels(pixels, pixel_count)) {
+  };
```

Comment:
in this case the semicolon can be removed as well.

Reply:
yeah, of course... I was sloppy in a rush again :) Will fix it now.